src/sandstorm/run-bundle.c++

```diff
@@ -1162,12 +1162,23 @@ private:
 
     // The environment inherited from the host is probably no good for us. E.g. an oddball
     // locale setting can crash Mongo because we don't have the appropriate locale files available.
+    //
+    // That said, there are a few environment variables that we do re-export.
+    char * http_proxy, * https_proxy;
+    KJ_SYSCALL(http_proxy = getenv("https_proxy"));
```

Comment:
`KJ_SYSCALL` only works for functions that return `int`. Also it's only meant for calls that can fail, but getenv() cannot fail (it returns null when there's no match, but that's not a failure).